examples/tracer-web/package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "web-tracer-example",
   "private": true,
-  "version": "0.25.0",
+  "version": "0.27.0",
   "description": "Example of using @opentelemetry/sdk-trace-web in browser",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server -d --progress --colors --port 8090 --config webpack.config.js --hot --inline --host 0.0.0.0 --content-base examples"
+    "start": "webpack serve --progress --color --port 8090 --config webpack.dev.config.js --hot --host 0.0.0.0 --compress",
```

Comment:
Some of the previous command line arguments don't work with v5 webpack, specifically the "-d" is the devtool mode, when not passing anything it fails to start.